infra/sentry/install_sentry.yml

```
@@ -0,0 +1,61 @@
+---
+- name: Install Sentry server
+  hosts: sentry
+  become: true
+  tasks:
+    - import_tasks: tasks/install_docker.yml
+      tags:
+        - 'docker'
+
+    - import_tasks: tasks/install_git.yml
+      tags:
+        - 'git'
+
+    - import_tasks: tasks/install_sentry.yml
+      tags:
+        - 'sentry'
+
+    - import_tasks: tasks/firewalld.yml
+      tags:
+        - 'firewalld'
+
+    - import_tasks: tasks/install_nginx.yml
+      tags:
+        - 'nginx'
+
+    - name: Check if dhparams file exists
+      stat:
+        path: /etc/ssl/certs/dhparam.pem
+      register: dhparams_file
+
+    - name: Generate  dhparams file
```

```suggestion
    - name: Generate  dhparams file if not exists
```

---

Je dirais même plus : 
```suggestion
    - name: Generate dhparams file if not exists
```